trk/src/Components/UserEdit.js

```diff
@@ -0,0 +1,206 @@
+    
+    //When the data loads, fetches the image of the latest climb to display for the session
+    useEffect(() => {
+        const loadImages = async () => {
+          try {
+            // Default image path
+            let climbImageURL = 'climb photos/the_crag.png';
+            if (user && user.image && user.image.length > 0) {
+                //Implement image fetch logic
+                climbImageURL = user.image[0].path;
+            }
+            const loadedClimbImageUrl = await loadImageUrl(climbImageURL);
+            setClimbImageUrl(loadedClimbImageUrl);
+            setInitialImagePath(loadedClimbImageUrl);
+          } catch (error) {
+            console.error("Error loading images: ", error);
+          }
+        };
+       loadImages();
+    }, [currentUser]);
+
+    const UpdateButton = ({ onPress }) => {
+        return (
+            <TouchableOpacity style={{width: '100%', backgroundColor: '#3498db', borderRadius: 10, display: 'flex', flexDirection: 'row', paddingVertical: 12, paddingHorizontal: 20, justifyContent: 'center', alignItems: 'center'}} onPress={onPress}>
+                <Text style={{color: 'white', fontSize: 15, textAlign: 'center'}}>Update Profile</Text>
+            </TouchableOpacity>
+        );
+    };
+
+    const uploadImage = async (imagePath) => {
+        const filename = imagePath.split('/').pop().replace(/\.jpg/gi, "").replace(/-/g, "");
+        const storageRef = storage().ref(`climb_images/${filename}`);
+        await storageRef.putFile(imagePath);
+        return { id: filename, path: storageRef.fullPath, timestamp: new Date().toISOString() };
+    };
+
+    const selectImageLogic = async () => {
+        console.log("handleImagePick called");
+        try {
+        const pickedImage = await ImagePicker.openPicker({
+            width: 200,
+            height: 200,
+            cropping: true,
+        });
+
+        // Save the full image path for later uploading
+        const imagePath = pickedImage.path;
+
+        // Set the image state to include the full path
+        setClimbImageUrl(imagePath);
+
+        } catch (err) {
+        console.error("Error picking image:", err);
+        }
+    };
+
+
+    const handleUpdateProfile = () => {
+        Alert.alert(
+            "Confirm Update",
+            "Are you sure you want to update your profile?",
+            [
+                {
+                    text: "Cancel",
+                    onPress: () => console.log("Update cancelled"),
+                    style: "cancel"
+                },
+                { 
+                    text: "OK", 
+                    onPress: () => updateProfile() 
+                }
+            ],
+            { cancelable: false }
+        );
+    };
+    
+    const updateProfile = async () => {
+        console.log('Initial image path: ', initialImagePath);
+        console.log('Updated image path: ', climbImageUrl);
+        console.log('Updating Profile for user: ', currentUser.uid);
```

Comment:
Seeing how you log variables to test their values, i'm realizing we should do a full cleanup of console logs on the last day of each sprint.

Reply:
Agreed! I've also been thinking that it becomes too cluttered at times. Will comment all out out from the next PR, and gradually take the older ones out too.